pkgs/servers/irc/solanum/default.nix

```diff
@@ -25,6 +25,10 @@ stdenv.mkDerivation rec {
     ./dont-create-logdir.patch
   ];
 
+  postPatch = ''
+    substituteInPlace include/defaults.h --replace 'ETCPATH "' '"/etc/solanum'
```

Comment:
Ugh, can't we configure the configuration path during runtime?

Reply:
Nope. The expected usage of this package is to install it to some folder in `/home/ircd` (typically `/home/ircd/solanum`) and then edit `ircd.motd` manually. It's a huge pain, but that's what happens when you have literal decades of cruft. :(

Reply:
Yeah, I remember looking at the code when we originally packaged it and it wasn't pretty.

Perhaps libredirect to the rescue?

Reply:
This particular path must be changed by setting --sysconfdir via configure (or this).  At present there does not exist a method to set the motd path at runtime (I do not know if this is intentional or historical, but my money is on historical).

Reply:
I'd prefer setting this in `configureFlags` then.

Reply:
I tried setting this in `configureFlags` and it didn't work.

Reply:
Please be a bit more precise as to what failed then.

Reply:
And a quick comment explaining the reasoning for the substitution would be appreciated.

Reply:
I don't know how to clarify "didn't work" further, the motd didn't show up until i added that flag